packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -99,6 +99,28 @@ const cli = {
           return undefined;
         },
       },
+      seleniumServiceUrl: {
+        type: 'string',
+        describe: 'The address for the selenium service.',
+        default: () => {
+          if (process.env.WDIO_SELENIUM_SERVICE_URL) {
+            return process.env.WDIO_SELENIUM_SERVICE_URL;
+          }
+
+          return undefined;
+        },
+      },
+      port: {
```

Comment:
We currently have `externalPort` as an existing option and now the name 'port' seems a little ambiguous. Can we change the name from `port` to `seleniumServicePort` to avoid any confusion?

Reply:
+1, sorry about that i forgot externalPort existed.